### DISCUSSION TITLE
CollectionView :(

### DISCUSSION THREAD

**Discussion by am-kh** [dotnet/maui#22327] [Q&A]
Of course, here's a refined version:

"Hello everyone,

Has anyone successfully implemented a collection view with complex cells in MAUI? The current performance of the CollectionView control is quite poor.

I have a list of 30 items containing images and labels (similar to what you might find on Booking or Trivago), and the performance issues are significant.

It's surprising to see MAUI struggling so much in this regard; it's making it nearly impossible to consider for production use.

PS: I've experimented with Uno Platform, and it delivers a very smooth experience. However, it lacks compatibility with several libraries, which is unfortunate."

**Comment by francoistanguay** [comment]
For what it's worth, you can use MAUI Embedding in Uno and use pretty much any MAUI library in an Uno app: https://platform.uno/net-maui-embedding/

**Comment by bradencohen** [comment]
Would you mind sharing your XAML? We use CollectionView pretty frequently and definitely don't see choppy performance at 20-30 items.

**Comment by MohamedSherifm** [comment]
I suggest using syncfusion collection view, until the collection view issues get fixed.

**Reply by burrowj** [reply]
As far as i know there, there is no syncfusion collection view, only the listview.  Is that not correct?
